modules/mongodb/mongodb.go

```diff
@@ -176,17 +185,27 @@ func initiateReplicaSet(req *testcontainers.GenericContainerRequest, cli mongoCl
 		req.LifecycleHooks, testcontainers.ContainerLifecycleHooks{
 			PostStarts: []testcontainers.ContainerHook{
 				func(ctx context.Context, c testcontainers.Container) error {
-					ip, err := c.ContainerIP(ctx)
+					// Wait for MongoDB to be ready
+					if err := waitForMongoReady(ctx, c, cli); err != nil {
+						return fmt.Errorf("failed to wait for MongoDB ready: %w", err)
```

Comment:
suggestion: "failed" prefix causes hard to read errors when wrapped. General rule is to describe what failed and then report the error for example:
```suggestion
						return fmt.Errorf("wait for mongo: %w", err)
```

More below.

Reply:
Removed "failed" prefix in wrapped errors.